docs/AppService-dotnetcore.md

```diff
@@ -53,11 +56,11 @@ Run the Validation Test
 
 ```bash
 
-# run the tests in the container
-docker run -it --rm retaildevcrew/webvalidate --server https://${He_Name}.azurewebsites.net --files helium.json
+# run the tests in a container
+docker run -it --rm retaildevcrew/webvalidate --server $He_App_Endpoint --base-url https://raw.githubusercontent.com/retaildevcrews/helium/main/TestFiles/ --files baseline.json
```

Comment:
I wasn't able to get helium.json to work, hence I changed this to use the baseurl instead.